.github/workflows/label-needs-triage.yml

```diff
@@ -12,5 +12,4 @@ jobs:
       - name: add labels
         uses: actions/labeler@v3
         with:
-          github_token: ${{ secrets.GH_AZURE_SDK_PROJECT_PAT }}
```

Comment:
@xiangyan99 you can develop this in your fork. I highly recommend going that direction.

1. Open `actions` in your fork. Enable them if necessary.
2. Update your `main` or default branch. 
3. Create issue.

Everything should work, and you will be able to do mul;tiple attempts without roundtripping in upstream main.

Reply:
hah, yes, this was what I wanted to say. I could not open issues in my fork.

What I can do is merge the PR in my fork and submit some changes.